mycroft/skills/main.py

```diff
@@ -607,6 +607,7 @@ def handle_converse_request(self, message):
                         "skill_id": 0, "result": False}))
                     return
                 try:
+                    LOG.info("INSTANCE: "+str(skill_id)+","+str(instance))
```

Comment:
Was this an intentional addition. In case it's intentional I think it's better to keep it at debug level.

Reply:
Yeah, I should probably keep that one local.  It is there to track down an issue I'm seeing where a skill_id with not instance is in there after a reload.